src/scene/turret/MissileT1.gd

```diff
@@ -1 +1,26 @@
 extends "res://scene/turret/Turrets.gd"
+
+var missile1_node;
+var missile2_node;
+var isLeftMissileFired = false;
+
+signal fire_missile(base_position, enemy, damage);
+
+func _ready():
+  ._ready();
```

Comment:
Does this need to be called at the end of the function? What does this do?

Reply:
This calls the parent's (`Turrets.gd`) `_ready()` function, which currently sets up the range of the turret.